script/upgradedb

```diff
@@ -75,26 +75,16 @@ if ($user) {
 my $schema = OpenQA::Schema::connect_db(check => 0);
 
 my $script_directory = "$FindBin::Bin/../dbicdh";
-my @databases        = qw( SQLite PostgreSQL );
-
-if ($schema->dsn =~ /:SQLite:dbname=(.*)/) {
-    # Some SQLite update scripts do not work correctly with foreign keys on
-    $schema->storage->dbh_do(
-        sub {
-            my ($storage, $dbh, @args) = @_;
-            $dbh->do("PRAGMA foreign_keys = OFF;");
-        });
-}
+my @databases        = qw( PostgreSQL );
```

Comment:
why you need array for single string ?

Reply:
because strings aren't arrays, hashes are, arrays aren't